examples/C/H5D/CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ if (BUILD_TESTING)
     endif ()
   endmacro ()
 
+  # Remove everything after "-"
```

Comment:
We should move this into a common macro/function in the config/cmake, since it repeats in every file.